organizations/tests.py

```diff
@@ -11,8 +11,30 @@ def setUp(self):
         Organization.objects.create(name="Org 1", shortcode="org-1")
         Organization.objects.create(name="Org 2", shortcode="org-2")
 
+    def test_guest_cannot_list_organization(self):
```

Comment:
isn't this already tested by `test_non_superuser_cannot_list_organizations`?

Reply:
Non superusers can be valid users (regular users, org-admins etc). Guests are unauthenticated users.